src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,14 @@
+@import "../utils/variables";
+
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.main__catalog {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 50px 40px;
+  width: $catalog-width;
```

Comment:
I I know U made a function, did U use it?

Reply:
Well fuction is useless here because result should be too specific for re-using. This way is much easier.